kube_scheduler/tests/test_sli_metrics.py

```diff
@@ -0,0 +1,87 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+
+import mock
+import pytest
+import requests
+import requests_mock
+
+from datadog_checks.kube_scheduler import KubeSchedulerCheck
+
+instance = {'prometheus_url': 'http://localhost:10259/metrics'}
```

Comment:
Instead of a custom `instance` here, you can use the `instance()` fixture from [here](https://github.com/DataDog/integrations-core/blob/324ec77d62ed9fa81c6a3373c94cc53cd5389898/kube_scheduler/tests/conftest.py#L21). You would need to update the `prometheus_url` value though.